llvm/docs/NVPTXUsage.rst

```diff
@@ -661,6 +661,126 @@ all bits set to 0 except for %b bits starting at bit position %a. For the
 '``clamp``' variants, the values of %a and %b are clamped to the range [0, 32],
 which in practice is equivalent to using them as is.
 
+'``llvm.nvvm.prmt``' Intrinsic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Syntax:
+"""""""
+
+.. code-block:: llvm
+
+    declare i32 @llvm.nvvm.prmt(i32 %a, i32 %b, i32 %c)
```

Comment:
I'd suggest more descriptive names for the arguments: %a->%lo, %b->%hi, %c -> %selector

Reply:
Fixed